internal/xds/bootstrap/jwtcreds/call_creds_test.go

```diff
@@ -0,0 +1,162 @@
+/*
+ *
+ * Copyright 2025 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package jwtcreds
+
+import (
+	"context"
+	"encoding/json"
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+	"time"
+
+	"google.golang.org/grpc/credentials"
+)
+
+func TestNewCallCredentials(t *testing.T) {
```

Comment:
Please use the `Tester` defined in the `grpctest` package here: https://github.com/grpc/grpc-go/blob/d0ebcdffc75dc76f18966ab9cccafe6a949d6fb5/internal/grpctest/grpctest.go#L52

It has some setup and teardown magic that we find useful for all of our tests.